gunrock/app/sm/sm_app.cu

```diff
@@ -17,15 +17,14 @@ namespace gunrock {
 namespace app {
 namespace sm {
 
-template <typename ParametersT>
-cudaError_t UseParameters(ParametersT &parameters) {
+cudaError_t UseParameters(util::Parameters &parameters) {
   cudaError_t retval = cudaSuccess;
   GUARD_CU(UseParameters_app(parameters));
   GUARD_CU(UseParameters_problem(parameters));
   GUARD_CU(UseParameters_enactor(parameters));
   GUARD_CU(UseParameters_test(parameters));
 
-  GUARD_CU(parameters.Use<unsigned int>(
+  GUARD_CU(parameters.template Use<unsigned int>(
```

Comment:
Sorry for the misunderstanding, you don't need `.template` if you switch it back to the old version. But if it compiles, you can leave this in there.

Reply:
The CI passed but I can change it back if you think it's necessary.